src/components/ui/TimeLineChart.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types'
+import React, {PureComponent} from 'react'
+import {Line} from 'react-chartjs-2'
+
+export default class TimeLineChart extends PureComponent {
+	static propTypes = {
+		data: PropTypes.object.isRequired,
+	}
+
+	render() {
+		const options = {
```

Comment:
Should probably be either a const outside the class or a member of the class instead of being re-eval'd every render step (potentially causing a rerender down the line because it's a new instance).

Reply:
mm, point